configure

```diff
@@ -287,9 +287,9 @@ check_vorbis()
 	fi
 }
 
-check_libsystemd()
+check_logind()
 {
-	pkg_config LIBSYSTEMD "libsystemd"
+	pkg_config LOGIND "libsystemd" || pkg_config LOGIND "libelogind >= 239.3"
```

Comment:
Can we only leave this bit in the patch? So that `LIBSYSTEMD_` stays etc.? I don't like having `elogind` names around.

Reply:
Yes, that should work.

Reply:
I just wanted to let you discuss this "internally" first, but I actually don't mind 😀

I'm not quite fond myself of the `LOGIND` change, but `LIBSYSTEMD` seemed even more generic, as only sd-bus is required. Anyway, pushing this!

Reply:
Why libelogind >= 239.3? Are earlier versions not compatible?

Reply:
@nefthy if I remember correctly version previous to 239.3 and 238.3 didn't have complete sd-bus support. I'm not sure how to correctly address this scenario.

Reply:
Yes, that's correct, but AFAIK, it should work fine quite a few versions back too.

Reply:
I'm OK with the check. Just wanted to make sure it is not arbitrary.